src/test/scala/is/hail/expr/ir/AggregatorsSuite.scala

```diff
@@ -1,5 +1,6 @@
 package is.hail.expr.ir
 
+import is.hail.expr._
```

Comment:
Why is this a file rename?

Reply:
It never had the right name. The class is called `AggregatorsSuite` but the file was called `AggregatorSuite.scala`

Reply:
I'm worried you overwrote a different test suite. Can you please double check?

Reply:
Are you referring to `src/test/scala/is/hail/methods/AggregatorSuite.scala`?  That's still there.